tests/TestCase/Core/Configure/Engine/PhpConfigTest.php

```diff
@@ -145,27 +147,28 @@ public function testDump()
         $this->assertTrue($result > 0);
         $expected = <<<PHP
 <?php
-return array (
-  'One' => 
-  array (
-    'two' => 'value',
-    'three' => 
-    array (
-      'four' => 'value four',
-    ),
-    'is_null' => NULL,
-    'bool_false' => false,
-    'bool_true' => true,
-  ),
-  'Asset' => 
-  array (
-    'timestamp' => 'force',
-  ),
-);
+return [
+    'One' => 
+    [
+        'two' => 'value',
+        'three' => 
+        [
+            'four' => 'value four'
+        ],
+        'is_null' => NULL,
```

Comment:
I wonder if we can lowercase those to "null" as per CS. Is the overhead to achieve this justified?